composer.json

```diff
@@ -22,20 +22,19 @@
 	],
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"php": ">= 7.4.0, <= 8.2.99",
+		"php": ">= 8.0.0, <= 8.2.99",
 		"ext-curl": "*",
 		"ext-json": "*",
-		"typo3/cms-core": "^11",
+		"typo3/cms-extbase": "^11 || ^12",
```

Comment:
Oh... one may now install TYPO3 (v12) without Extbase? funny, I can't think of it :)

Reply:
Nope, but it is not correct to require only core, when we use functionality from other system exts too.
I scanned all usages (hopefully) and adjusted accordingly.
I dropped core here, because the more specific sysexts require it anyways

Reply:
fine